### PR TITLE
Added my ROM TrollogenMOD

### PR DESCRIPTION
**Pull request by TrollDevTeam**

### CODE REVIEW

**Comment by ctso** [comment]
Pretty funny, but you seriously think we're going to merge this?  Stop wasting our time.

Thanks!

**Comment by koush** [comment]
LOL @ctso, I think it's a legit mod?

**Comment by koush** [comment]
http://forum.xda-developers.com/showthread.php?t=1059424&page=14

**Comment by ctso** [comment]
Yeah I got called out via PM after trashing it.  Real trolling would be removing all builds, except TrollogenMOD, apparently.

**Comment by koush** [comment]
Remove your trailing comma after vision, and fix your formatting please.

**Comment by TrollDevTeam** [comment]
Damn only read trailing comma and ingnored the rest XD...

File: manifests.js [hunk 1]
```diff
@@ -1159,7 +1159,18 @@
...
-        }		
+        }	
+        {
+	"developer": "Troll Dev Team",
+	"free": "true",
+	"id": "DanWilson",
+	"icon": "http://img651.imageshack.us/img651/9139/trollfaced.png",
+	"manifest": "http://dl.dropbox.com/u/7700554/TrollogenMOD.js",
+	"roms": {
+			"vision": "true",
```
Comment (judeibe) [inline manifests.js:1170]: @trolldevteam looks like the problem is right here `"vision": "true",`  should be `"vision": "true"`

**Comment by TrollDevTeam** [comment]
Yeah I've also to fix formatting though...

**Comment by koush** [comment]
You need to update your pull request. I'll fix it myself I guess.

**Comment by TrollDevTeam** [comment]
Do I? Man I've never really understood github...
Cheers man. Its real appreciated.

**Pull request merged**